### DISCUSSION TITLE
Value Error ("The shape of mean and the standard deviation must be identical")  in attacks

### DISCUSSION THREAD

**Discussion by abhinav-vimal-17** [Trusted-AI/adversarial-robustness-toolbox#1954] [General]
1.    adv = ProjectedGradientDescent(classifier, targeted=False, max_iter=10, eps_step=1, eps=5)
2.   adv_img = adv.generate(adv_in_img), where adv_in_img is a numpy array

In the second line, I am getting Value Error ("The shape of mean and the standard deviation must be identical")

**Comment by beat-buesser** [comment]
Hi @abhinav-vimal-17 How did you define the mean and the standard deviation in the definition of the classifier?
